### ISSUE TITLE
Please update the draw.io to newest version. the vscode client is not same as web client. :(

### ISSUE THREAD

**Issue by Tairraos** [hediet/vscode-drawio#444]
Please update the draw.io to newest version. the vscode client is not same as web client. :(

thanks a looooooooooooot.

**Comment by jdillard** [comment]
You can use the latest (online version) by following these instructions: https://github.com/hediet/vscode-drawio/issues/171#issuecomment-1120326817

**Comment by hediet** [comment]
Will update soon!

**Comment by JakeGinnivan** [comment]
Happy new year @hediet :) Just wondering if you have had a chance to look at a nice fresh release of this extension. The recent issues around right click stopping working are hitting me as well and I'm hoping an update just fixes that too.

**Comment by davidjgraph** [comment]
Hi @hediet https://github.com/hediet/vscode-drawio/pull/451 is by one of the draw.io team.

**Comment by hediet** [comment]
Sorry for the delay, I was very busy the last months and finally had some long vacation :)
I did some clean-up and just published `1.7.2025012805` as pre-release version (deprecating the insiders-build).
I'd be very happy if you could test that release a bit, I'll do a stable release for the same commit in a couple of days!

**Comment by JakeGinnivan** [comment]
@hediet glad you had a well deserved break. 

I just updated to the latest VSCode and pre-release of the extension and #450 is now fixed as well. I'll report back if I notice any new issues but at first glances seems to be ok.

**Comment by cbluth** [comment]
@hediet thanks for a wonderful extension 🎉  its amazing

for the update, i tested it 
```
Name: Draw.io Integration
Id: hediet.vscode-drawio
Description: This unofficial extension integrates Draw.io into VS Code.
Version: 1.7.2025012806
Publisher: Henning Dieterichs
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio
```
```
Version: 1.96.4
Commit: cd4ee3b1c348a13bafd8f9ad8060705f6d4b9cba
Date: 2025-01-16T00:16:19.038Z
Electron: 32.2.6
ElectronBuildId: 10629634
Chromium: 128.0.6613.186
Node.js: 20.18.1
V8: 12.8.374.38-electron.0
OS: Darwin arm64 23.5.0
```

i cannot see the context-menu on rightclick as anticipated, instead it grabs/pans
also, a new bug that i am experiencing, i cannot change the theme at the bottom, it seems whatever i select it will stay in a dark theme, when i typically use the lighter theme kennedy, changing the theme has not effect

**Comment by JakeGinnivan** [comment]
Yeah actually, looks like my VSCode was still on the rolled back version. My bad, #450 still exists
